docs/README.md

```diff
@@ -62,20 +56,17 @@ will also install the [self-node-remediation] operator as a default remediator.
 For development environments you can run `make deploy deploy-snr`.
 See the [Makefile](./../Makefile) for more variables.
 
-On start the controller creates a default resource name `nhc-worker-default`,
-that remediates using Self Node Remediation, and will check for worker-only heath issues.
-See [NodeHealthCheck Custom Resource](#nodehealthcheck-custom-resource) for the default properties.
-If there is an existing resource the controller will not create a default one.
-
 ## NodeHealthCheck Custom Resource
 
-Here is the default NHC resource the operator creates on start:
+The Node Healthcheck operator requires configuration by creating one or more
+NHC custom resources. An example CR for remediating worker nodes with the
+Self Node Remediation operator looks like this:
 
 ```yaml
 apiVersion: remediation.medik8s.io/v1alpha1
 kind: NodeHealthCheck
 metadata:
-  name: nhc-worker-default
+  name: nhc-snr-worker
```

Comment:
NIT: Why associate the CR name with SNR? We might want to decouple NHC from SNR.
How about `nhc-worker` or `nhc-worker-remediator`?

Reply:
because it uses SNR?

Reply:
I am aware that ATM it uses SNR, and I was thinking/looking on the future when NHC won't be installed with SNR. 
We can also cross the bridge once we reach to that future point 😄

Reply:
Seriously? Why do you worry about SNR being a dependency in the future? This config uses SNR. And I don't see the smallest reason to not have snr in its name.

Reply:
I wasn't looking on the whole PR properly... Especially this [line](https://github.com/medik8s/node-healthcheck-operator/pull/174/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R63). 
There is no need to use `remediator` or similar. Using `snr` indeed makes sense.